drapps/README.md

```diff
@@ -0,0 +1,110 @@
+# DRApps
+
+## About 
+
+DRApps is simple CLI tool for creating your custom application in DataRobot.
+
+Custom applications does not provide any storage but you may access the full DataRobot API and other Internet services
+
+## Installation
+
+For using this CLI script you need only install python requirements with:
+
+    pip install -r requirements.txt
+
+## Usage
+
+    $ ./drapps.py --help
+    Usage: drapps.py [OPTIONS]
+    
+      App that uses local file for create new custom application
+    
+    Options:
+      -e, --base-env TEXT   Name or ID for execution environment  [required]
+      -p, --path DIRECTORY  Path to folder with files that should be uploaded
+                            [required]
+      -n, --name TEXT       Name for new custom application. Default: CustomApp
+      -t, --token TEXT      Pubic API access token.
+      -E, --endpoint TEXT   Data Robot Public API endpoint
```

Comment:
Cosmetic nit but still important IMO, anywhere that mentions `Data Robot` should be changed to `DataRobot`